### PR TITLE
wpsoffice: 11.1.0.9080 -> 11.1.0.9126

### PR DESCRIPTION
**Pull request by Ninlives**
###### Motivation for this change

Update wpsoffice

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

# Help wanted

The application generally works, but with following problems:
1. Open `Settings` and `Skin Center`, which will execvp `./result/opt/kingsoft/wps-office/office6/promecefpluginhost`, silently fails and exit.
2. Using `wpp` to open some `.ppt` files crashes without error messages;
3. `et` does not display CJK words correctly.

### CODE REVIEW

File: pkgs/applications/office/wpsoffice/default.nix [hunk 1]
```diff
@@ -77,18 +62,31 @@ stdenv.mkDerivation rec{
     cp -r usr/* $out
     # Avoid forbidden reference error due use of patchelf
     rm -r *
-    for i in wps wpp et wpspdf; do
-      patchelf \
-        --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-        --force-rpath --set-rpath "$(patchelf --print-rpath $prefix/office6/$i):${stdenv.cc.cc.lib}/lib64:${libPath}" \
-        $prefix/office6/$i
+    bins=$(find $prefix/office6 -maxdepth 1 -executable ! -type d|grep -v '.*\.so\.*')
```
Comment (veprbl) [inline pkgs/applications/office/wpsoffice/default.nix]: Looks like you are reinventing `autoPatchelfHook` here https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/auto-patchelf.sh

**Comment by veprbl** [comment]
Just a side note: When you do the bump and a refactoring at the same time, it makes it harder to review the diff.

**Comment by Th0rgal** [comment]
Duplicate of: https://github.com/NixOS/nixpkgs/pull/84438

**Comment by Ninlives** [comment]
Close in favor of https://github.com/NixOS/nixpkgs/pull/85211

**Pull request closed without merging**